types/xmlbuilder/index.d.ts

```diff
@@ -85,5 +85,5 @@ declare class XMLElementOrXMLNode {
 }
 
 declare namespace xmlbuilder {
-    function create(name: string, xmldec?: Object, doctype?: any, options?: Object): XMLElementOrXMLNode;
+    function create(nameOrObjSpec: string | { [name:string]: Object }, xmldec?: Object, doctype?: any, options?: Object): XMLElementOrXMLNode;
```

Comment:
It could, but each value which is not of type string or object with a key/value leads to function `create()` throwing an exception.